stock_orderpoint_move_link/models/procurement_group.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ProcurementGroup(models.Model):
+    _inherit = "procurement.group"
+
+    @api.model
+    def run(self, product_id, product_qty, product_uom, location_id, name,
+            origin, values):
+        if 'orderpoint_id' in values:
+            if values['orderpoint_id'].name not in origin:
+                origin += ', %s' % values.get('orderpoint_id').name
```

Comment:
it is possible for origin to be empty? If so this could end being something like `, OP/NNN` with an initial comma that may be unwanted.

Reply:
It's hard to get it, action who triggers this is commonly https://github.com/odoo/odoo/blob/11.0/addons/stock/models/stock_move.py#L762,
I can add more code to avoid this if required, I don't think so.

Reply:
comments attended